### PR TITLE
 Created schema for faq and category table

### PR DESCRIPTION
**Pull request by RubiTTN**
Changes:

1. Created schema for FAQ
2. Created schema for category
3. Created enum for faq type
4. Create mutation to add a new faq
5. Create query to fetch all faq or a faq based on id
6. Create mutation to update faq

### CODE REVIEW

File: backend/prisma/datamodel.prisma [hunk 1]
```diff
@@ -9,6 +9,11 @@ enum PageType {
   ARTICLE
 }
 
+enum FaqType {
+  FAQ
```
Comment (sumandomarpa) [inline backend/prisma/datamodel.prisma]: We don't need `FAQ type`. We have to resolve the `tooltip` in different module.

File: backend/prisma/datamodel.prisma [hunk 2]
```diff
@@ -74,4 +79,28 @@ type Cons {
...
+  variant: [String]
+  tag: [String]
+  pubDate: DateTime
+}
+
+type FaqCategory {
+  id: ID! @unique
+  name: String!
+  slug: String!
+  description: String!
```
Comment (sumandomarpa) [inline backend/prisma/datamodel.prisma]: We have to resolve Many to Many relationship between Faq and FaqCategory.

File: backend/prisma/datamodel.prisma (hunk 1 above)
Comment (RubiTTN) [reply backend/prisma/datamodel.prisma]: Sure, Removing.

File: backend/prisma/datamodel.prisma (hunk 2 above)
Comment (RubiTTN) [reply backend/prisma/datamodel.prisma]: @sumandomarpa Added many to many relationship between faq and categories.

File: backend/prisma/datamodel.prisma [hunk 3]
```diff
@@ -76,6 +76,30 @@ type Cons {
   order: Int
 }
 
+type Faq {
+  id: ID! @unique
+  title: String!
+  description: String
+  short_description: String
```
Comment (sumandomarpa) [inline backend/prisma/datamodel.prisma]: Do we need `short_description`?

File: backend/prisma/datamodel.prisma [hunk 4]
```diff
@@ -76,6 +76,30 @@ type Cons {
...
+  authors: [User!]!
```
Comment (sumandomarpa) [inline backend/prisma/datamodel.prisma]: Why there is many authors for a single Faq?

File: backend/prisma/datamodel.prisma [hunk 5]
```diff
@@ -76,6 +76,30 @@ type Cons {
...
+  slug: String!
+  vertical: String
+  category: [FaqCategory!]!
+  readingTime: Int
+  order: Int
+  variant: [String]
```
Comment (sumandomarpa) [inline backend/prisma/datamodel.prisma]: What is `variant`?

File: backend/prisma/datamodel.prisma (hunk 3 above)
Comment (RubiTTN) [reply backend/prisma/datamodel.prisma]: I took this reference from existing word-press application.

File: backend/prisma/datamodel.prisma (hunk 4 above)
Comment (RubiTTN) [reply backend/prisma/datamodel.prisma]: In existing word-press application there are option to add multiple authors. That's why I added authors as an array.

File: backend/prisma/datamodel.prisma (hunk 5 above)
Comment (RubiTTN) [reply backend/prisma/datamodel.prisma]: I took this reference from existing word-press application.

**Pull request merged**